src/workspaces/list/NoContentMessage.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { screen } from '@testing-library/react';
+import React from 'react';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import { defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+import { CategorizedWorkspaces } from 'src/workspaces/list/CategorizedWorkspaces';
+import { NoContentMessage } from 'src/workspaces/list/NoContentMessage';
+import { WorkspaceFilterValues } from 'src/workspaces/list/WorkspaceFilters';
+
+describe('NoContentMessage', () => {
+  const defaultWorkspace: CategorizedWorkspaces = {
+    myWorkspaces: [defaultGoogleWorkspace],
+    newAndInteresting: [],
+    featured: [],
+    public: [],
+  };
+
+  const noWorkspaces: CategorizedWorkspaces = {
+    myWorkspaces: [],
+    newAndInteresting: [],
+    featured: [],
+    public: [],
+  };
+
+  const keywordFilters: WorkspaceFilterValues = {
+    keywordFilter: 'nothingMatchesThis',
+    accessLevels: [],
+    tab: 'myWorkspaces',
+    tags: [],
+  };
+
+  const featuredFilters: WorkspaceFilterValues = {
+    keywordFilter: '',
+    accessLevels: [],
+    tab: 'featured',
+    tags: [],
+  };
+
+  it('displays loading message if workspaces are loading', () => {
+    // Arrange
+    const props = { loadingWorkspaces: true, workspaces: defaultWorkspace, filters: keywordFilters };
+
+    // Act
+    render(<NoContentMessage {...props} />);
+
+    // Assert
+    expect(screen.getByText('Loading...')).toBeInTheDocument();
+  });
+
+  it('displays no workspaces message if there are no workspaces and filters tab is myWorkspaces', () => {
+    // Arrange
+    const props = { loadingWorkspaces: false, workspaces: noWorkspaces, filters: keywordFilters };
```

Comment:
Nit: might not want to use `keywordFilters` here. The point is that you are on the myWorkspaces tab, and there are no workspaces to display. The keyword part is not relevant for this test.

In general, in tests it is more acceptable (and sometimes preferable) to duplicate test setup instead of trying to reuse variables a lot because it makes it clearer what the test is expecting. I wouldn't mind seeing the filters variable just defined inline here.